backend/src/schema/resolvers/searches.spec.js

```diff
@@ -477,6 +614,30 @@ und hinter tausend Stäben keine Welt.`,
           })
         })
       })
+
+      describe('searchPostQuery', () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-22T13:14:36Z" title="Monday, June 22nd 2020, 3:14:36 pm +02:00">Jun 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -477,6 +614,30 @@ und hinter tausend Stäben keine Welt.`,
           })
         })
       })
+
+      describe('searchPostQuery', () => {
```


Thanks again for the new test! 😍